app/utils/helpers.rb

```diff
@@ -13,12 +13,13 @@ def is_time?(t)
   false
 end
 
-def open_now?(place, user_now, place_now)
+def open_now?(opening_hours, user_now, place_now)
   # This method returns true if the user_now (time object) is between the
   # open and close hours of the place. The place_now determines the place's
   # day to match the hours.
+  # opening_hours is a hash of hours of a place
   day = place_now.strftime('%A')[0...3].downcase
```

Comment:
Day == first 3 letters of the day?
"Saturday" -> "Sat"?
[apidocks](https://apidock.com/ruby/DateTime/strftime)
suggestion: why not use 
`place_now.strftime(%a).downcase`

```
Weekday:
  %A - The full weekday name (``Sunday'')
          %^A  uppercased (``SUNDAY'')
  %a - The abbreviated name (``Sun'')
          %^a  uppercased (``SUN'')
  %u - Day of the week (Monday is 1, 1..7)
  %w - Day of the week (Sunday is 0, 0..6)

```